### DISCUSSION TITLE
rust-course/关于格式化输出新增特性

### DISCUSSION THREAD

**Discussion by allandowney** [sunface/rust-course#441] [内容建议]
1.58 版本新增的 `在格式化字符串时捕获环境中的值` 的内容是否要加入？

**Comment by sunface** [comment]
我记得以前有补充，但是刚看了下，内容并不存在，奇了怪了。。。

**Comment by sunface** [comment]
已添加。。。感谢，下次你也可以直接通过 pr 的方式提交建议，我觉得你的建议都很好 :)
